contracts/token/ERC721/ERC721.sol

```diff
@@ -57,7 +57,7 @@ contract ERC721 is ERC165, IERC721 {
      * @return uint256 representing the amount owned by the passed address
      */
     function balanceOf(address owner) public view returns (uint256) {
-        require(owner != address(0));
+        require(owner != address(0), "empty or zero transaction recipient");
```

Comment:
There's no transaction here, just a query. Maybe something like `balance query for zero address`?

Reply:
I will change it.